docs/overview/ern-workflow.md

```diff
@@ -30,7 +30,7 @@ This command is equivalent the the Yarn `add` or `npm install` commands.
 - Launch the MiniApp inside the Electrode Native platform runner mobile application.  
 The runner allows you to develop, debug, and test the MiniApp standalone, using the `ern run-android` and/or the `ern run-ios` commands. These commands are the equivalent of the `react-native run-android` and `react-native run-ios` commands.  
 
-If you are a MiniApp developer who just wants to open source a MiniApp to be used by any mobile application using Electrode Native, this will probably be the only workflow stage that you'll be involved in. If you want maximum reach for your MiniApp, you should also make sure to update the versions of the native dependencies your MiniApp uses--to align with the dependencies declared in the Electrode Native master manifest at every platform release (every 2 weeks on Monday). This can easily be achieved using the Electrode Native `ern upgrade` command. You should ideally have one branch for each Electrode Native version ideally (we don't have automated support for this, but might come at some point to easily align many MiniApps on a selected Electrode Native version).
+If you are a MiniApp developer who just wants to open source a MiniApp to be used by any mobile application using Electrode Native, this will probably be the only workflow stage that you'll be involved in. If you want maximum reach for your MiniApp, you should also make sure to update the versions of the native dependencies your MiniApp uses--to align with the dependencies declared in the Electrode Native master manifest at every platform release (every 2 weeks on Monday). This can easily be achieved using the Electrode Native `ern upgrade` command. You should ideally have one branch for each Electrode Native version (we don't have automated support for this, but might come at some point to easily align many MiniApps on a selected Electrode Native version).
```

Comment:
The change doesn't look obvious. But I only removed one word `ideally` from the sentence `You should ideally have one branch for each Electrode Native version ideally`.

Reply:
thanks for contributing. Unfortunately, as much as I'd like to, I can't merge it as of now. As for many other open source project (react-native included), you'll have to sign a CLA first (for legal reasons) before we can accept your code contributions ... We are currently working on setting-up this process properly, to make it easier for people to contribute. Will keep you posted.

Reply:
@krunalsshah or anyone else who works at Walmart Labs, would you guys mind letting me know when you have the CLA process set up? I would love to contribute to Electrode Native.